static/docs/commands-reference/remote:add.md

```diff
@@ -41,6 +42,13 @@ and optionally assigns a default remote in the core section:
    remote = myremote
 ```
 
+DVC supports the concept of a _default remote_.  For the commands which take
```

Comment:
should we mention a few - (like `dvc pull`, `dvc gc`, etc)?

Reply:
Okay